frontend/javascripts/oxalis/shaders/texture_access.glsl.js

```diff
@@ -207,6 +207,14 @@ export const getColorForCoords: ShaderModule = {
 
       float rgbaIndex = linearizeVec3ToIndexWithMod(offsetInBucket, bucketWidth, packingDegree);
 
+      if (packingDegree == 2.0) {
+        if (rgbaIndex == 0.0) {
+          return vec4(bucketColor.r, bucketColor.g, bucketColor.r, bucketColor.g);
+        } else if (rgbaIndex == 1.0) {
+          return vec4(bucketColor.b, bucketColor.a, bucketColor.b, bucketColor.a);
+        }
+      }
+
```

Comment:
Maybe add a comment line here stating that the following code deals with packingDegree == 4.0? :) Took me a few seconds to figure out why there isn't a `if (packingDegree == 4.0) {` statement.